breathecode/events/tests/receivers/tests_generate_event_recap_on_finished.py

```diff
@@ -0,0 +1,39 @@
+import random
+from unittest.mock import MagicMock, call, patch
+
+from breathecode.admissions.models import CohortTimeSlot
+from breathecode.events.models import Event  # Import Event
+
+# TODO: Add necessary imports for models, receivers, etc.
+from breathecode.events.receivers import generate_event_recap_on_finished  # Corrected receiver name
+from breathecode.events.signals import event_status_updated
+from breathecode.events.tasks import generate_event_recap  # Import only the relevant task to mock
+
+from ..mixins.new_events_tests_case import EventTestCase
+
+
+class GenerateEventRecapOnFinishedTestSuite(EventTestCase):
+    """Tests for event_status_updated receiver that triggers generate_event_recap"""
+
+    @patch("breathecode.events.tasks.generate_event_recap.delay")
+    def test_event_status_updated_receiver__call_generate_recap_when_finished(self, generate_recap_mock):
+        """Check if generate_event_recap task is called when event status changes to FINISHED"""
+        event = self.bc.database.create(event=1)
+        event.event.status = "FINISHED"
+
+        generate_event_recap_on_finished(sender=None, instance=event.event, new_status="FINISHED", old_status="ACTIVE")
```

Comment:
`new_status` and `old_status` is not sended by `event_status_updated` nor used by `generate_event_recap_on_finished`

Reply:
Prefer pytest style if possible